backend/src/main/java/lt/techin/lsf/model/requests/UpdateUserRequest.java

```diff
@@ -0,0 +1,41 @@
+package lt.techin.lsf.model.requests;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import jakarta.validation.constraints.Max;
+import jakarta.validation.constraints.Min;
+import jakarta.validation.constraints.NotBlank;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.Setter;
+
+@Getter
+@Setter
+@AllArgsConstructor
+public class UpdateUserRequest {
+
+    @NotBlank(message = "Field cannot be empty")
+    @JsonProperty("name")
+    private String name;
+
+    @NotBlank (message = "Field cannot be empty")
+    @JsonProperty("surname")
+    private String surname;
+
+    @Min(value = 1900, message = "Birth year must be after 1900")
```

Comment:
zinutese vietoje datos siulau rasyti {value}, tada reiksme keisis priklausomai nuo to kokia reikme irasyti prie value =.